src/private/doctor/doctorDashboard.html

```diff
@@ -22,11 +22,9 @@
         </noscript>
         <!-- GÓRNA BELKA I MENU -->
         <main class="main">
-            <div class="col card hoverable s10 pull-s1 m8 pull-m2 l8 pull-l2"
-                style="padding:16px; margin: 24px 4px 4px 24px; ">
+            <div class="col card hoverable s10 pull-s1 m8 pull-m2 l8 pull-l2" style="padding:16px; margin: 24px 4px 4px 24px; ">
                 <div class="center" style="padding: 0px; margin: 0px;">
-                    <lottie-player src="https://assets9.lottiefiles.com/temp/lf20_vS6iGX.json" background="transparent"
-                        speed="1" style="width: 200px; height: 200px;" loop autoplay>
+                    <lottie-player src="https://assets9.lottiefiles.com/temp/lf20_vS6iGX.json" background="transparent" speed="1" style="width: 200px; height: 200px;" loop autoplay>
```

Comment:
Nie wiem jaki sens tych zmian, bo używane było autoformatowanie przeze mnie i mieściło się bez scrollowania na ekranie. Ręcznie to formatowałaś, czy inaczej Ci formatuje IDE?

Reply:
Z automatu się zmieniło to formatowanie